packages/storybook8/stories/Concepts/AdhocCalling/Docs.mdx

```diff
@@ -0,0 +1,65 @@
+import { MessageBar, MessageBarType } from '@fluentui/react';
+import { Meta, Source } from '@storybook/addon-docs';
+
+import AdhocCallingSnippetText from '!!raw-loader!./snippets/AdhocCalling.snippet.tsx';
+import AdhocCallingVoiceAppSnippetText from '!!raw-loader!./snippets/AdhocCallingVoiceApp.snippet.tsx';
+
+<Meta title="Concepts/Ad-hoc calling" />
+
+# Ad-hoc calling
+
+## Overview
+
+A call can be started with a Teams user or a call queue by using their id. You can call a teams user or a Teams voice application
+like a call queue or auto attendant. These calls it is important to note can only be one Teams voice application, or multiple teams users. You cannot call multiple
+Teams voice applications at the same time.
+
+## Calling a Teams voice application
+
+To call a Teams voice application like a call queue or auto attendant, you need to use the id of the Teams resource account. Check out this [tutorial](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-teams-auto-attendant#find-object-id-for-auto-attendant) on how to find
+this id. The id of the Teams resource account should be stored in the type of `MicrosoftTeamsAppIdentifier[]` and given to the `targetCallees` property
+of the [CallAdapter](?path=/docs/composite-adapters--page#call-adapter) when creating a call. Now when the [CallComposite](?path=/docs/composites-call-basicexample--basic-example) starts up it will call the Teams voice application
+from the configuration screen. See the following snippet for an example.
+
+<MessageBar messageBarType={MessageBarType.warning}>
+  Note: Teams resource account ids linked to a call queue need to be in format '28:orgid:UUID' when used in the
+  CallParticipantsLocator. For example, '28:orgid:87d349ed-44d7-43e1-9a83-5f2406dee5bd'.
+</MessageBar>
+
+<Source code={AdhocCallingVoiceAppSnippetText} />
+
+The two callback functions `onFetchProfile` and `onFetchAvatarPersonaData` provided at the bottom of the snippet are
+for customizing display names and avatars of users, respectively.
+
+## Calling a Teams user
+
```

Comment:
Based on storybook 6 it looks like you are missing a beta banner here
![image](https://github.com/user-attachments/assets/523787d2-b936-4203-ab58-73f40a369f73)

Reply:
I removed the beta banner since ad-calling is not in stable.

Reply:
Not in stable? Or is in stable?

Reply:
Sorry typo, it is **now** in stable